extensions/sql-database-projects/package.json

```diff
@@ -119,36 +139,78 @@
         {
           "command": "sqlDatabaseProjects.newFolder",
           "when": "false"
+        },
+        {
+          "command": "sqlDatabaseProjects.build",
+          "when": "false"
+        },
+        {
+          "command": "sqlDatabaseProjects.deploy",
+          "when": "false"
+        },
+        {
+          "command": "sqlDatabaseProjects.import"
+        },
+        {
+          "command": "sqlDatabaseProjects.properties",
+          "when": "false"
         }
       ],
       "view/item/context": [
         {
-          "command": "sqlDatabaseProjects.close",
-          "when": "view == sqlDatabaseProjectsView"
+          "command": "sqlDatabaseProjects.build",
+          "when": "view == sqlDatabaseProjectsView",
+          "group": "1_dbProjectsFirst"
         },
         {
-          "command": "sqlDatabaseProjects.newScript",
-          "when": "view == sqlDatabaseProjectsView"
+          "command": "sqlDatabaseProjects.deploy",
+          "when": "view == sqlDatabaseProjectsView",
+          "group": "1_dbProjectsFirst"
+        },
+        {
+          "command": "sqlDatabaseProjects.import",
+          "when": "view == sqlDatabaseProjectsView",
+          "group": "1_dbProjectsFirst"
+        },
+        {
+          "command": "sqlDatabaseProjects.newItem",
+          "when": "view == sqlDatabaseProjectsView",
+          "group": "2_dbProjects_newMain@1"
+        },
+        {
+          "command": "sqlDatabaseProjects.newFolder",
+          "when": "view == sqlDatabaseProjectsView",
+          "group": "2_dbProjects_newMain@2"
         },
         {
           "command": "sqlDatabaseProjects.newTable",
-          "when": "view == sqlDatabaseProjectsView"
+          "when": "view == sqlDatabaseProjectsView",
+          "group": "3_dbProjects_newItem@1"
         },
         {
           "command": "sqlDatabaseProjects.newView",
-          "when": "view == sqlDatabaseProjectsView"
+          "when": "view == sqlDatabaseProjectsView",
+          "group": "3_dbProjects_newItem@2"
         },
         {
           "command": "sqlDatabaseProjects.newStoredProcedure",
-          "when": "view == sqlDatabaseProjectsView"
+          "when": "view == sqlDatabaseProjectsView",
+          "group": "3_dbProjects_newItem@3"
         },
         {
-          "command": "sqlDatabaseProjects.newItem",
-          "when": "view == sqlDatabaseProjectsView"
+          "command": "sqlDatabaseProjects.newScript",
+          "when": "view == sqlDatabaseProjectsView",
+          "group": "3_dbProjects_newItem@9"
         },
         {
-          "command": "sqlDatabaseProjects.newFolder",
-          "when": "view == sqlDatabaseProjectsView"
+          "command": "sqlDatabaseProjects.close",
+          "when": "view == sqlDatabaseProjectsView",
+          "group": "9_dbProjectsLast"
```

Comment:
why does this start with 9?

Reply:
the number is the order of the group in the context menu; I wanted to indicate that anyone adding new commands that this should stay last, and new groups should be added in before it.